zrml/authorized/src/lib.rs

```diff
@@ -118,8 +156,12 @@ mod pallet {
         MarketDoesNotHaveDisputeMechanismAuthorized,
         /// An account attempts to submit a report to an undisputed market.
         MarketIsNotDisputed,
+        /// Only one dispute is allowed.
+        OnlyOneDisputeAllowed,
         /// The report does not match the market's type.
         OutcomeMismatch,
+        /// The market should be reported at this point.
+        MarketIsNotReported,
```

Comment:
This seems to be never used.

Reply:
Thanks Vivek! b41af6ac3e873f08706b5c584f7bb9a86e65a542